### ISSUE TITLE
[2.55] NPE backing up game

### ISSUE THREAD

**Issue by tchegito** [tchegito/zildo#171]
```
java.lang.NullPointerException: 
  at zildo.monde.Game.serialize (Game.java:132)
  at zildo.client.gui.menu.SaveGameMenu.saveGame (SaveGameMenu.java:165)
  at zildo.client.gui.menu.SaveGameMenu.saveOnCrash (SaveGameMenu.java:179)
  at com.alembrum.OpenGLRenderer.onDrawFrame (OpenGLRenderer.java:121)
  at android.opengl.GLSurfaceView$GLThread.guardedRun (GLSurfaceView.java:1522)
  at android.opengl.GLSurfaceView$GLThread.run (GLSurfaceView.java:1239)
```

**Comment by tchegito** [comment]
It happens when game has already crashed (most likely with issue #170 which happened with or withour hero in the logs). Then we try to back up the game, although no hero is present.

In this case, we shouldn't try to back up.

**Comment by tchegito** [comment]
[Fixed in 2.56]

Now, we don't try to back up if hero isn't present yet.

7e11f4799f140c3e4dc4a0deaad574b1f3c3c839
